kafka-rest/src/assembly/package.xml

```diff
@@ -47,14 +47,15 @@
             <excludes>
                 <exclude>io.confluent:rest-utils</exclude>
                 <exclude>org.slf4j:slf4j-log4j12</exclude>
```

Comment:
not-a-comment-to-be-addressed: In theory isn't this a redundant leftover? I would expect `slf4j-log4j12` to be "extinct" from our dependencies no matter what packaging is being used. In practice it probably makes sense to keep this exclusion until it can be proven to be redundant.

Reply:
Yeah, I think it is safe to keep it here for now.